app/market/item/[id]/page.jsx

```diff
@@ -1,43 +1,136 @@
+          <Link color="inherit" href="/" onClick={() => router.push('/')}>
+            Home
+          </Link>
+          <Typography color="text.primary">Card Details</Typography>
+        </Breadcrumbs>
+
+        {/* Image and Details Section */}
+        <div style={{ display: 'flex', marginTop: theme.spacing(2)}}>
+          {/* Image Section */}
+          {cardDetails && (
+            <Card style={{ boxShadow: 'none', marginRight: theme.spacing(2) }}>
+              <CardActionArea type="button" onClick={() => router.push(`/market/item/${cardDetails.id}`)}>
+                <CardMedia
+                  component="img"
+                  image={cardDetails.imageURL}
+                  alt={cardDetails.name}
+                  style={{ width: 300, height: "auto" }}
+                />
+              </CardActionArea>
+            </Card>
+          )}
+
+          {/* Details Section */}
+          {cardDetails && (
+            <Box style={{ maxWidth: 600, paddingLeft: theme.spacing(2), borderRadius: theme.shape.borderRadius }}>
+              <Typography variant="h4" gutterBottom>
+                {cardDetails.name}
+              </Typography>
+              <Typography variant="subtitle1" gutterBottom>
+                Price: {getCurrencySymbol(cardDetails.currency)}{cardDetails.price}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Description: {cardDetails.description}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Conditions: {cardDetails.conditions}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Category: {cardDetails.category}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Quantity: {cardDetails.quantity}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Availability: {cardDetails.available}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Set: {cardDetails.set}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                Shipping Cost: {cardDetails.shippingCost}
+              </Typography>
+            </Box>
+          )}
+        </div>
+
+        {/* Action Buttons */}
+        <div style={{ display: 'flex', gap: theme.spacing(1), marginTop: theme.spacing(2) }}>
+          <Button variant="contained" color="accent" onClick={handleAddToCartButtonClick} style={{ color: theme.palette.background.paper }}>
+            <ShoppingCartIcon />
+            Add to cart
+          </Button>
+
+          <Button variant="contained" color="primary" onClick={handleEditButtonClick}>
+            Edit card
+          </Button>
+          <Button variant="contained" color="secondary" onClick={handleSellerInfoButtonClick}>
```

Comment:
What do you think: instead of this button we can add the seller name on the card information? Like on Amazon page: 
<img width="328" alt="image" src="https://github.com/Code-the-Dream-School/ffprac-team6/assets/106414157/e3652d71-3f35-452d-b277-db8c25ebe02a">
the name should be clicable. In this case we will not use too manu buttons?